gui/cppcheck_de.ts

```diff
@@ -5,39 +5,46 @@
     <name>About</name>
     <message>
         <location filename="about.ui" line="14"/>
+        <location filename="build-cppcheck-Desktop-Debug/gui/ui_about.h" line="130"/>
```

Comment:
@firewave  this is wrong. the filename should not be "build-cppcheck-Desktop-Debug/gui/ui_about.h". In my opinion these ts changes should be reverted. Please cleanup these changes of the ts files or else we remove that ts files are generated automatically. I don't want to keep cleaning up the mess manually anymore.

Reply:
You did still not drop these changes.

Reply:
Your comment, “You did still not drop these changes,” came across to me as harsh. I’ve been against generating the TS files automatically because I anticipated it would disrupt our workflow, and I’m tired of repeatedly cleaning them up. If the cleanup isn’t handled moving forward, we will need to disable the TS file generation.